release_management/_utils.py

```diff
@@ -38,8 +38,17 @@ class ReleaseVersion:
     """
 
     def __init__(self, base, suffix):
+        """
+        Initializer.
+        :param str base: Base version
+        :param str or NoneType: Version suffix
+        """
         self.base = base
-        self.suffix = suffix
+
```

Comment:
This should work fine. Being more inclined toward functional programming, I would usually write the same thing as:

```
self.suffix = "" if suffix is None else suffix
```

It's a bit shorter and makes it very obvious that ```self.suffix``` has _some_ value assigned to it, after that line is executed.

Reply:
Resolved.